crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1444,6 +1440,7 @@ impl Client {
             .send(SessionChange::UnknownToken { soft_logout: *soft_logout });
     }
 
+    /// Fetches server capabilities from network; no caching.
```

Comment:
Err, that's a very confusing doc for a method called `fetch_and_cache_...`.

Reply:
yep lol